wordpress-mobile/WordPress-iOS
# Notifications + Swift Mk 8: View Lifecycle

#### Details:
This PR adds no new functionality. We'll gradually migrate all of the Notifications-Y code over to Swift.
Once it's ready, we'll address code improvements / technical debt.

Reference #5613

--

#### How to Test:
- Please, verify that the Notifications List shows up properly
- Rotate the view and check that it works as expected
- Verify that the filters work as they should
- Try logging out and into a different account
- Try opening an Automattcher Notification. The reader should show up onscreen
- Try opening a Comment-Y Notification, and validate that everything works as expected

@kwonye *ALMOST* there with **NotificationsViewController**.
Ready for review sir!!

Thanks in advance!!
